Tests/TestMessagesViewControllerModel.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  TestMessagesViewControllerModel.swift
+//  MessageKit
+//
+//  Created by Dan Leonard on 8/14/17.
+//  Copyright © 2017 MessageKit. All rights reserved.
+//
```

Comment:
Hey @legalimpurity I update the readme link to not expire . Its this: https://join.slack.com/t/messagekit/shared_invite/MjI4NzIzNzMyMzU0LTE1MDMwODIzMDUtYzllYzIyNTU4MA